Doc/distutils/apiref.rst

```diff
@@ -4,6 +4,16 @@
 API Reference
 *************
 
+.. seealso::
+
+   `New and changed setup.py arguments in setuptools <setuptools-setup-py>`_
```

Comment:
@ncoghlan This is a dead link to https://docs.python.org/3.8/distutils/setuptools-setup-py

See linkchecker output:

```

URL        `setuptools-setup-py'
Name       `New and changed setup.py arguments in setuptools'
Parent URL file:///builddir/build/BUILD/Python-3.8.0b4/Doc/build/html/distutils/apiref.html, line 106, col 5
Real URL   file:///builddir/build/BUILD/Python-3.8.0b4/Doc/build/html/distutils/setuptools-setup-py
Check time 0.001 seconds
Result     Error: URLError: <urlopen error [Errno 2] No such file or directory: '/builddir/build/BUILD/Python-3.8.0b4/Doc/build/html/distutils/setuptools-setup-py'>
 4 threads active,     0 links queued, 4445 links in 4449 URLs checked, runtime 1 minute, 46 seconds
```

Where is this actually supposed to lead to?